docs/zh/latest/terminology/plugin.md

```diff
@@ -65,8 +65,6 @@ local _M = {
 plugins:
   - real-ip                   # 安装
   - ai
-  - real-ip
-  - ai
```

Comment:
The same issue exists in the en document.

Reply:
Hi @hummingg, could you also modify the English document section?

Reply:
Hi @hummingg, following up on the previous review comments. Please let us know if you have any updates. Thank you.